ui/src/overridden/frontsite/Document/DocumentDetails/DocumentMetadataTabs/Identifiers.js

```diff
@@ -0,0 +1,211 @@
+import {
+  InfoPopup,
+  SeparatedList,
+  vocabularyApi,
+  withCancel,
+} from '@inveniosoftware/react-invenio-app-ils';
+import capitalize from 'lodash/capitalize';
+import _isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Divider, Table, Loader } from 'semantic-ui-react';
+
+export class Identifiers extends Component {
+  constructor(props) {
+    super(props);
+
```

Comment:
could you maybe add a disclaimer that this component shouldn't be used in search pages?

Reply:
sure :+1:

Reply:
why exactly not again? 😅

Reply:
because it might trigger many requests for vocabs one for each row of the search result
even with redux now, we will mount the instances of this component and some of them might perform multiple request before the state gets filled with vocab data

Reply:
ah ok! makes sense 😄